__tests__/rojo-says.spec.js

```diff
@@ -0,0 +1,123 @@
+describe('rojoSays', () => {
+  let test;
+
+  describe('when sentense starts with word hello', () => {
+    beforeEach(() => {
+      test = {};
+      test.subject = require('../lib/rojo-says').listen;
+    });
+
+    it('should return a string containing "hello"', () => {
+      const helloRegex = /hello.*/i;
+      const testString = 'Hello there!'
+      const result = helloRegex.test(test.subject(testString));
+      expect(result).toBe(true);
+    });
+    describe('.listen', () => {
+      describe('when passed string that starts with "when"', () => {
+        it('should return a string containing am or pm', () => {
+          const timeRegex = /.+am|pm.*/i;
+          const testString = 'When are you coming home?';
+          const result = timeRegex.test(test.subject(testString));
+          expect(result).toBe(true);
```

Comment:
Once you get better with tests, the preferred pattern is to put setup in a beforeEach, and then your it block will only contain assertions. Example:

```js
beforeEach(() => {
  test.regex = /.*am|pm*/i;
  test.result = test.subject('When are you coming home?');
});

it('should return a string containing am or pm', () => {
  expect(test.regex.test(test.result)).toBe(true);
});
```